### PR TITLE
[DOCS] update README.md @open sesame 06/18 13:33

### PR DESCRIPTION
**Pull request by EunjuYang**
This PR contains commits to
- add recent NNTrainer paper and its citation to `README.md`
- fix trivial typo in `README.md`

### CODE REVIEW

**Review by skykongkong8 — APPROVED** [review]
Now we got it right ;) LGTM

**Review by DonghakPark — APPROVED** [review]
LGTM!!!

**Review by baek2sm — APPROVED** [review]
it looks good to add citation info. LGTM!

File: README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ NNtrainer is an Open Source Project. The aim of the NNtrainer is to develop a So
 
 Even if NNtrainer runs on device, it provides full functionalities to train models and also utilizes limited device resources efficiently. NNTrainer is able to train various machine learning algorithms such as k-Nearest Neighbor (k-NN), Neural Networks, Logistic Regression, Reinforcement Learning algorithms, Recurrent network and more. We also provide examples for various tasks such as Few-shot learning, ResNet, VGG, Product Rating and more will be added. All of these were tested on Samsung Galaxy smart phone with Android and PC (Ubuntu 18.04/20.04).
 
+[ A New Frontier of AI: On-Device AI Training and Personalization ](https://dl.acm.org/doi/abs/10.1145/3639477.3639716), ICSE-SEIP, 2024 <br />
 [ NNTrainer: Light-Weight On-Device Training Framework ](https://arxiv.org/pdf/2206.04688.pdf), arXiv, 2022 <br />
 [ NNTrainer: Towards the on-device learning for personalization ](https://www.youtube.com/watch?v=HWiV7WbIM3E), Samsung Software Developer Conference 2021 (Korean) <br />
```
Comment (djeong20) [inline README.md:23]: this link is currently invalid. maybe we can also update it

**Review by djeong20 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (djeong20) [inline README.md:23]: this link is currently invalid. maybe we can also update it
```suggestion
[  Open Source On-Device AI SW Platform ](https://youtu.be/im3uNrPLYx4?si=gMbw7LKKSnpXi59U), Samsung Developer Conference 2023 (Korean) <br />
```

**Comment by myungjoo** [comment]
Merge when @djeong20 's comment is applied.

**Review by myungjoo — APPROVED** [review]

**Review by SeoHyungjun — APPROVED** [review]
LGTM

**Pull request merged**